src/ValidatorForm.jsx

```diff
@@ -99,6 +99,7 @@ class ValidatorForm extends React.Component {
             validateResult = this.constructor.getValidator(validator, value, includeRequired);
             result.push({ input, result: validateResult });
             component.validate(component.props.value, true, dryRun);
+            component.makeTouched();
```

Comment:
why component should be touched after validation?

Reply:
@NewOldMax it doesn't needed to be untouched, since the input looses its virginity after typing

Reply:
But it will be touched not only after typing. Validation may call programmatically, or only after clicking `submit` button. Anyway, what kind of flexibility do you need? Can you provide example?

Reply:
when the input is null it becomes valid again, in those cases i need it to show errors @NewOldMax

Reply:
Sorry, it's not clear for me. Can you provide real code example? on codesandbox

Reply:
in angular js we have ng-touched but not in react if this feature is added it will improve in the future for many scenarios @NewOldMax